build/Prepare.targets

```diff
@@ -13,6 +13,14 @@
                             CheckPrereqs;">
   </Target>
 
+  <Target Name="_SandBoxPrepare"
```

Comment:
NIT: We don't release need the underscore on this target name.

Reply:
Fixed, not _blah in this PR